.github/workflows/build_and_test_debug.yml

```diff
@@ -90,59 +90,6 @@ jobs:
           path: build/dist/Outline-Client.exe
           if-no-files-found: error
 
-  mac-debug-build:
```

Comment:
Please restore macOS. It's fixed by https://github.com/Jigsaw-Code/outline-client/pull/1256

Reply:
Just did